content/cpp-from-scratch.md

```diff
@@ -0,0 +1,124 @@
+An introduction to m3pi programming
```

Comment:
Note: this was written last-minute for a session, we should rewrite it ASAP to make it non-terrible. However, for the time being, it is significantly better than nothing.